Sources/BrowserServicesKit/ContentBlocking/ContentBlockerRulesSourceManager.swift

```diff
@@ -71,6 +71,22 @@ public class ContentBlockerRulesSourceModel: ContentBlockerRulesSourceIdentifier
  Manages sources that are used to compile Content Blocking Rules, handles possible broken state by filtering out sources that are potentially corrupted.
  */
 public class ContentBlockerRulesSourceManager {
+    
+    public class RulesSourceBreakageInfo {
+
+        public let name: String
```

Comment:
@bwaresiak the name doesn't seem to be used by anything at the moment, can it be removed or do you have other plans for it?